dotCMS/src/integration-test/java/com/dotmarketing/portlets/fileassets/business/FileAssetAPITest.java

```diff
@@ -170,5 +173,161 @@ public void Test_That_File_Asset_Gets_Stored_in_Cache_and_is_Not_Rebuilt_Everyti
       assertTrue("FileAssets should be the same Object", asset3 == asset4);
       
     }
+    
+    /**
+     * This tests that file assets saved to a random folder, e.g. /dasfasd/ will be returned from the database
+     * @throws Exception
+     */
+    @Test
+    public void test_that_file_asset_from_db_works()
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6325390)